apis/quay/v1/quayregistry_types.go

```diff
@@ -83,10 +83,13 @@ const (
 
 // QuayRegistrySpec defines the desired state of QuayRegistry.
 type QuayRegistrySpec struct {
-	// ConfigBundleSecret is the name of the Kubernetes `Secret` in the same namespace which contains the base Quay config and extra certs.
+	// ConfigBundleSecret is the name of the Kubernetes `Secret` in the same namespace
+	// which contains the base Quay config and extra certs.
 	ConfigBundleSecret string `json:"configBundleSecret,omitempty"`
 	// Components declare how the Operator should handle backing Quay services.
 	Components []Component `json:"components,omitempty"`
+	// Overrides holds overrides for the Base component (quay-app).
+	Overrides *Override `json:"overrides,omitempty"`
```

Comment:
Not sure why this line is showing as an addition. You might have to rebase?

Reply:
This `Override` is for the `base` component (there is no such thing as `base` in the list of components so I put it in the spec's root). I am not happy with it but I could not think in any other approach (to add a `base` component to `spec.components` I think is a no go as it can't be "unmanaged"). Do you have any other ideas?

Reply:
`base` = `quay-app`? Why can't it be unmanaged?

Reply:
It can't be "unmanaged" without creating a ton of problems for us at this stage (mirror will crash is one example). I have created a component "base" but forced it to be managed (rolls back if user sets it to "unmanaged"). To support such a component as "unmanaged" is a totally different task and I won't be addressing it here.

Reply:
I will be opening a new Jira Card to deal with this "unmanaged base component" thing. Resolving this conversation so we can get this in.